src/pages/MetricDetail.svelte

```diff
@@ -2,10 +2,126 @@
   import { getMetricData } from "../state/api";
 
   export let params;
+
   const metricDataPromise = getMetricData(params.app, params.metric);
 </script>
 
+<style>
+  .table-header {
+    @apply table-auto;
+    @apply my-4;
+  }
+  .table-header td {
+    @apply border;
+    @apply p-2;
+  }
+</style>
+
 {#await metricDataPromise then metric}
   <h1>{metric.name}</h1>
   <p>{metric.description}</p>
+  <table class="table-header">
+    <tr>
+      <td>Relevant Bugs</td>
+      <td>
+        {#each metric.bugs as bug, i}
+          {#if bug.indexOf('http') > -1}
+            <a href={bug} title={bug} target="_blank"> {i + 1} </a>
+          {:else}<span>{bug}</span>{/if}
+        {/each}
+      </td>
+    </tr>
+    <tr>
+      <td>Send In Pings</td>
+      <td>{metric.send_in_pings.join(', ')}</td>
```

Comment:
Can you make this a link to the actual ping views? e.g. http://localhost:5000/#!/apps/burnham/pings/discovery

Reply:
Here , for few metrics under fenix application like "about_page.libraries_tapped" the ping value is "events" 
But I dont see fenix/pings/events as a valid json
Nor do I see "events" in the list of pings under fenix : http://localhost:5000/data/fenix/tables/events.json is nonexistent

Please guide me on what I might be missing here :)
Thanks.

Reply:
You actually stumbled on a missing feature: we're not processing dependencies for our applications. See (just filed) #51 

For now you can test with this metric:

http://localhost:5000/#!/apps/fenix/metrics/activation.activation_id

The found in pings should have the activation ping, and the URL you provide should take you to:

http://localhost:5000/#!/apps/fenix/pings/activation